xml/target_types_mrw.xml

```diff
@@ -655,6 +655,19 @@
 			<id>FABRIC_CHIP_ID</id>
 			<default>0</default>
 		</attribute>
+	</targetType>
+	<targetType>
+		<id>module-module-turismo</id>
+		<parent>card</parent>
+		<parent_type>socket-proc_socket-50mm</parent_type>
+		<attribute>
+			<id>TYPE</id>
+			<default>NA</default>
+		</attribute>
+		<attribute>
+			<id>PROC_PCIE_IOP_CONFIG</id>
```

Comment:
I need unique attributes for the naples socket. Please ignore this pull request and I will create another 50mm socket target for naples and move these attributes back to socket-proc_socket-50mm